tailwind.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     mode: 'jit',
     purge: [],
     darkMode: false, // or 'media' or 'class'
+    important: true,
```

Comment:
Due to this config may affected all site of css, we cannot guarantee that all page is correct.

I don't think it's a good idea to add important in every tailwind generated css.

If we're going to migrate scss to tailwind, then maybe we should have a migration plan or we should review the new code and refactor to tailwind in every new PRs of new code.